.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Please describe your changes -->
+
+## How Has This Been Tested?
+<!-- Please describe in detail how you tested your changes. -->
+<!-- Include details of your testing environment, tests ran to see how -->
+<!-- your change affects other areas of the code, etc. -->
+
+## Screenshots (jpeg or gifs if applicable):
+
+## Types of changes
```

Comment:
I've founds screenshots to be very helpful, especially gifs, when reviewing a feature or UI fix/enhancement. Also, normal contributors don't have access to label our PRs.